arrow-libs/core/arrow-core/src/main/kotlin/arrow/core/extensions/option/apply/OptionApply.kt

```diff
@@ -70,16 +72,16 @@ fun <A, B> Kind<ForOption, A>.apEval(arg1: Eval<Kind<ForOption, Function1<A, B>>
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-    "zipEval(arg1) { b, c -> arg2(Tuple2(b, c)) }",
-    "arrow.core.Tuple2",
-    "arrow.core.zipEval"
+    "arg1.map { fb -> fix().zip(fb.fix()) { a, b -> arg2(Tuple2(a, b)) } }",
+    "arrow.core.fix"
```

Comment:
```suggestion
    "arrow.core.fix",
    "arrow.core.zip"
```

Reply:
`zip` is concrete on `Option` so no import required.